test/gtest/uct/test_flush.cc

```diff
@@ -401,6 +403,7 @@ void uct_flush_test::test_flush_am_pending(flush_func_t flush, bool destroy_ep)
      }
 
      if (destroy_ep) {
+        ASSERT_FALSE(is_flush_cancel());
```

Comment:
should be other way around
if we flush with cancel - destroy_ep must be true
otherwise - destroy_ep could be either true or false